src/Uno.UI.Tests/Windows_UI_Xaml_Data/BindingTests/Given_Binding.cs

```diff
@@ -198,5 +200,21 @@ public void When_Xaml_Object_With_Xaml_Object_Properties()
 			binding.RelativeSource.Mode.Should().Be(RelativeSourceMode.None);
 			binding.Source.Should().Be("Source");
 		}
+
+		[TestMethod]
+		public void When_Parent_Set_To_Null()
+		{
+			using var _ = new AssertionScope();
+			var panel = new StackPanel();
+			var tb = new TextBox();
+			tb.SetBinding(TextBox.IsEnabledProperty, new Binding());
```

Comment:
Note to self: this test shouldn't use IsEnabledProperty, because it shouldn't be available on StackPanel and have been moved in breaking changes. We can use another inherited property like `AllowFocusWhenDisabledProperty`